api4/system.go

```diff
@@ -27,6 +28,7 @@ func (api *API) InitSystem() {
 
 	api.BaseRoutes.ApiRoot.Handle("/audits", api.ApiSessionRequired(getAudits)).Methods("GET")
 	api.BaseRoutes.ApiRoot.Handle("/email/test", api.ApiSessionRequired(testEmail)).Methods("POST")
+	api.BaseRoutes.ApiRoot.Handle("/site_url/test", api.ApiSessionRequired(testSiteURL)).Methods("POST")
```

Comment:
Just a reminder: please also submit docs at https://github.com/mattermost/mattermost-api-reference

Reply:
Thanks for the reminder. Here it is: https://github.com/mattermost/mattermost-api-reference/pull/469